### PR TITLE
chore: Show disabled Team Health option to Starter tier

### LINKED ISSUE #8663: Show Team Health to Starter users
See @garethaledavies's message with his idea [here](https://parabol.slack.com/archives/C03LL1JRFH8/p1692036162902079) and   discussion [here](https://parabol.slack.com/archives/C042E8PGPD4/p1692036615963339).

### AC

- [ ] Show Team Health in the activity library settings for Starter users
- [ ] It should be disabled for Starter users and have a greyed out UI
- [ ] When hovering over it, a tooltip explains that it's a premium feature

### PR DESCRIPTION
**Pull request by Dschoordsch**
# Description

Fixes #8663

## Demo

Screenshot from 2023-08-16 10-53-50
Screenshot from 2023-08-16 10-56-17

## Testing scenarios

- [ ] check both start meeting dialogs with a Starter team
- [ ] check both start meeting dialogs with a Team team

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by Dschoordsch** [comment]
Having a CTA in the tooltip does not really work. We could redesign the button instead, sth. like this:
Screenshot from 2023-08-17 13-15-56
@acressall what do you think?

**Comment by acressall** [comment]
I like this change! Here's some small design tweaks https://www.figma.com/file/pOUIVgle3xwzo4OSf1fCZG/Flow---Team-Health-Check?type=design&node-id=201-1291&mode=design&t=F5TsyKQNcCTPZUgv-0
Let's have the upgrade link open in a new tab

disabled-option

**Comment by Dschoordsch** [comment]
https://www.loom.com/share/4f2464492bcd49a38d8205fde8e753c6?sid=258dba3d-7b67-4b04-9df4-82e1ccdfec59

File: packages/client/components/NewMeetingSettingsUpgradeForTeamHealth.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  const {orgId} = team
+  const atmosphere = useAtmosphere()
+
+  const handleUpgrade = () => {
+    SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
+      upgradeCTALocation: 'meetingSettingsTeamHealth',
+      meetingType: 'retrospective'
+    })
+    window.open(`/me/organizations/${orgId}/billing`, '_blank', 'noreferrer')
```
[the remark this answers is not in the document]
Comment (tianrunhe) [reply packages/client/components/NewMeetingSettingsUpgradeForTeamHealth.tsx:53]: LGTM here! Can we also add a `Upgrade CTA Viewed` event to complete the funnel?

**Review by tianrunhe — APPROVED** [review]
LGTM from metrics side!

**Review by igorlesnenko — APPROVED** [review]
Looks good, I would slightly tweak paddings, maybe centering the whole text would help, not critical

image

**Comment by Dschoordsch** [comment]
Text is not centered in the designs, so I kept it like this. I think with the smaller subtext that's ok.

**Pull request merged**